### DISCUSSION TITLE
支付宝购买专业版账号依然无法恢复问题

### DISCUSSION THREAD

**Discussion by taoistx** [getlantern/lantern-forum-cn#438] [General]
@Mike777Khoo 
你好，账号注册邮箱是 taoistx@163.com，距离上次反馈已有一周时间，现在进度如何？
如果无法处理，请尽快退款

**Comment by taoistx** [comment]
已解决，感谢

**Reply by MaxMKMax** [reply]
@taoistx 好的. 不客气. 谢谢您支持蓝灯!
